pkg/validation/errors/schema.go

```diff
@@ -181,6 +182,7 @@ func TooManyProperties(name, in string, n int64) *Validation {
 		Name:    name,
 		In:      in,
 		Value:   n,
+		Valid:   n,
```

Comment:
change the name of n to make it clear what it should be (maxProperties), and make sure callers are setting it as expected